pac4j-cas/src/main/java/org/pac4j/cas/client/CasClient.java

```diff
@@ -266,7 +266,7 @@ protected void initializeClientConfiguration() {
             this.casPrefixUrl += "/";
         }
         if (CommonHelper.isBlank(this.casPrefixUrl)) {
-            this.casPrefixUrl = this.casLoginUrl.replaceFirst("/login", "/");
+            this.casPrefixUrl = this.casLoginUrl.replaceFirst("/login$", "/$");
```

Comment:
dollar in order to match just the end of the regex. Was that the original idea of the line or are there any other legitimate replacement cases?